stubs/chevron/chevron/renderer.pyi

```diff
@@ -1,9 +1,13 @@
 from _typeshed import StrPath, SupportsRead
 from collections.abc import MutableSequence, Sequence
 from typing import Any
+from typing_extensions import Literal
 
 g_token_cache: dict[str, list[tuple[str, str]]]  # undocumented
+linesep: str
```

Comment:
This is just a re-export from `os`; we shouldn't include it in the stub, I don't think. It would be a code smell for people to do `from chevron.renderer import linesep` instead of `from os import linesep`, and it would be weird for `linesep` to start popping up in IDE autocompletes. This should just be allowlisted.